packages/workspace-utils/package.json

```diff
@@ -22,6 +22,7 @@
     "@prairielearn/path-utils": "workspace:^",
     "@prairielearn/postgres": "workspace:^",
     "@socket.io/redis-emitter": "^5.1.0",
+    "execa": "^5.1.1",
```

Comment:
Are we going to stick with calling `du` or switch to your JS version? I thought the JS version was nice for avoiding compatibility issues with different `du` implementations, and for allowing us to be clear about what we are computing.

Reply:
Yeah, I can buy that we want to be really explicit about what we're computing and how we're doing so. I'll change this to a JS implementation.